src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -170,10 +172,28 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   fileValidity = false
   customErrorMessage = ['', '', '', '', ''] // max 5 files
 
+  get minAuthorizationDate (): string {
+    // the BC Founding Date (if it exists, ie, expro business)
+    // else the Home Jurisdiction Incorporation Date (ie, manual entry)
+    // else fall back to null (not undefined)
+    return (
+      this.getExistingBusinessInfo.bcFoundingDate ||
+      this.getExistingBusinessInfo.homeIncorporationDate ||
+      null
+    )
+  }
```

Comment:
Fall back to null because 'undefined' is not reactive (ie, getters and watchers won't recognize its change).